TypeCobol/Compiler/CupParser/EmptyTypeCobolProgram.cup

```diff
@@ -1032,6 +1033,17 @@ jsonGenerateStatementConditional ::=
 		{:  :}
 	;
 
+jsonParseStatementConditional ::=
```

Comment:
`jsonParseStatementConditional` should be added in the non terminal list and in the `compoundStatement` production rule, like you did in TypeCobolProgram.cup.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1542/commits/70bde4b647a278978958313a518d6f507e1083c5